executor/aggfuncs/aggfunc_test.go

```diff
@@ -975,37 +783,96 @@ func testAggMemFunc(t *testing.T, p aggMemTest) {
 	}
 }
 
-// Deprecated: migrating to testAggMemFunc(t *testing.T, p aggMemTest)
-func (s *testSuite) testAggMemFunc(c *C, p aggMemTest) {
-	srcChk := p.aggTest.genSrcChk()
+func testMultiArgsAggFunc(t *testing.T, ctx sessionctx.Context, p multiArgsAggTest) {
```

Comment:
We need to reuse a context object in `TestGroupConcat` test.  `func (s *testSuite) testMultiArgsAggFunc(c *C, p multiArgsAggTest)` has been reusing context via `testSuite` object, but this function has not receiver object, so i change how to reuse context via function argment.